universal-login-sdk/test/api/wallet/DeployingWallet.test.ts

```diff
@@ -55,6 +55,7 @@ describe('UNIT: DeployingWallet', () => {
   it('waits to be success', async () => {
     const deployedWallet = await deployingWallet.waitToBeSuccess();
     const {deploymentHash, ...serializedDeployedWallet} = serializedDeployingWallet;
+    ensure(deployedWallet instanceof DeployedWallet, Error, 'variable should be instance of DeplloyedWallet');
```

Comment:
Why not use `InvalidWalletState`?

Reply:
Because it just a sanity check for typescript. I want to throw a specific error for the test. And this error gives us complete information about the test's error reason